### PR TITLE
Add scribl.

### PR DESCRIPTION
**Pull request by jneem**

### CODE REVIEW

**Comment by jneem** [comment]
Scribl is a program for creating little videos. The repository is [here](https://github.com/jneem/scribl)

**Comment by nedrichards** [comment]
bot, build ink.scribl.Scribl

File: ink.scribl.Scribl.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    buildsystem: simple
+    build-commands:
+      - source /usr/lib/sdk/rust-stable/enable.sh && cargo build --release
+      - install -D target/release/scribl /app/bin/scribl
+      - install -D metadata/ink.scribl.Scribl.appdata.xml /app/share/metainfo/ink.scribl.Scribl.appdata.xml
+      - install -D metadata/ink.scribl.Scribl.svg /app/share/icons/scalable/apps/ink.scribl.Scribl.svg
+      - install -D metadata/ink.scribl.Scribl.desktop /app/share/applicatons/ink.scribl.Scribl.desktop
+    build-options:
+      build-args:
+        - --share=network
```
Comment (bilelmoussaoui) [inline ink.scribl.Scribl.yml:34]: The builders don't have network access during the build stage. The sources must be downloaded before; please use https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo

File: ink.scribl.Scribl.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  # Permission to write to app-specific configuration and cache directories.
+  - --filesystem=xdg-config
+  - --filesystem=xdg-cache
+  - --filesystem=xdg-data
```
Comment (bilelmoussaoui) [inline ink.scribl.Scribl.yml:17]:
```suggestion

```
You should have access to those by default.

File: ink.scribl.Scribl.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: scribl
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (bilelmoussaoui) [inline ink.scribl.Scribl.yml:11]:
```suggestion
  - --socket=fallback-x11
```

**Comment by jneem** [comment]
Thanks for the review! I think the latest patch addresses the issues. Now let's see if the build bot will listen to me...

**Comment by jneem** [comment]
bot, build ink.scribl.Scribl

**Comment by bilelmoussaoui** [comment]
The appdata file validation seems to fail due to 
```
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
• tag-missing           : <release> required
```

**Comment by jneem** [comment]
bot, build ink.scribl.Scribl

**Comment by jneem** [comment]
bot, build ink.scribl.Scribl

File: ink.scribl.Scribl.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            },
+            "generated-sources.json"
+        ],
+        "buildsystem": "simple",
+        "build-commands": [
+            "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+            "cargo --offline build --release --verbose",
+            "install -D target/release/scribl /app/bin/scribl",
+            "install -D metadata/ink.scribl.Scribl.appdata.xml /app/share/metainfo/ink.scribl.Scribl.appdata.xml",
+            "install -D metadata/ink.scribl.Scribl.svg /app/share/icons/scalable/apps/ink.scribl.Scribl.svg",
```
Comment (bilelmoussaoui) [inline ink.scribl.Scribl.json:35]:
```suggestion
            "install -D metadata/ink.scribl.Scribl.svg /app/share/icons/hicolor/scalable/apps/ink.scribl.Scribl.svg",
```

Comment (jneem) [reply ink.scribl.Scribl.json:35]: Thanks! I also added some pre-rendered pngs, since the build seemed to complain about that.

**Comment by jneem** [comment]
bot, build ink.scribl.Scribl

**Comment by jneem** [comment]
bot, build ink.scribl.Scribl

**Pull request closed without merging**